Products/ZenHub/zenhub.py

```diff
@@ -204,6 +207,8 @@ def perspective_reportingForWork(self, worker):
         @return None
         """
         worker.busy = False
+        if pid is not None:
```

Comment:
When this code is reached, `worker` is a [remote reference](http://twistedmatrix.com/documents/8.2.0/api/twisted.spread.pb.RemoteReference.html), and so it won't have the pid, unless the actual `zenhubworker` sends it its `ZenHub` when it reports for work, or I'm able to associate the pid of newly spawned `zenhubworker` proc, with the corresponding worker in `ZenHub.workers`. `ZenHub` has three collections of objects referring to `zenhubworkers` (see [here](https://github.com/zenoss/zenoss-prodbin/blob/develop/Products/ZenHub/zenhub.py#L414-L423)). This was the clearest way to associate the correct hubworker pid with the worker, when the Hub is reporting statistics on that worker.